modules/dag-l1/src/main/scala/org/tessellation/dag/l1/infrastructure/trust/storage/TrustDbStorage.scala

```diff
@@ -0,0 +1,124 @@
+package org.tessellation.dag.l1.infrastructure.trust.storage
+
+import cats.effect.MonadCancelThrow
+import cats.syntax.functor._
+import cats.syntax.traverse._
+
+import org.tessellation.dag.l1.domain.trust.storage.TrustStorage
+import org.tessellation.dag.l1.infrastructure.db.Database
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.schema.trust._
+
+import doobie.implicits._
+import doobie.quill.DoobieContext
+import io.getquill.context.Context
+import io.getquill.{Literal, SqliteDialect}
+
+object TrustDbStorage {
+  case class StoredTrust(
+    peerId: PeerId,
+    score: Option[Score],
+    rating: Option[Rating],
+    observationAdjustment: Option[ObservationAdjustment]
+  )
+
+  type Ctx = DoobieContext.SQLite[Literal] with Context[SqliteDialect, Literal]
+
+  def make[F[_]: MonadCancelThrow: Database]: TrustStorage[F] =
+    make[F](new DoobieContext.SQLite[Literal](Literal))
+
+  def make[F[_]: MonadCancelThrow](ctx: Ctx)(implicit db: Database[F]): TrustStorage[F] =
+    new TrustStorage[F] {
+      val xa = db.xa
+
+      import ctx._
+
+      val getPeerIds = quote {
```

Comment:
Renamed to `getStoredTrust` and updated code accordingly. Also renamed the other functions to replace `PeerId` with `StoredTrust`.

Reply:
There is no need to add `getStored` prefix to every single method because we already know that you get that from storage. The case class is named `TrustDbStorage`. That decision may lead to redundant naming like `storage.getStoredTrust` instead of just `storage.getTrust`. Also the prefix "Trust" is redundant (`TrustScore`, `TrustRating`, etc) because we already know that it is `TrustStorage`. It would be fine if case class was named `TrustScore` but you named it just `Score` so I would go for `getScore` which will be called like this: `trustStorage.getScore`

TL:DR:

Just use `getTrust`, `getScore`, `getRating` without `...StoredTrust` prefix